components/delegations/cells/ContractCell.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+
+import AddressDisplay from 'components/address/AddressDisplay';
+import type { Delegation } from 'lib/delegate/DelegatePlatform';
+import { useTranslations } from 'next-intl';
+
+interface Props {
+  delegation: Delegation;
+}
+
+const ContractCell = ({ delegation }: Props) => {
+  const t = useTranslations();
+
+  if (!delegation.contract) {
+    return (
+      <td className="whitespace-nowrap py-4 pl-4 pr-3 text-sm sm:pl-6">
+        <span className="text-gray-500 dark:text-gray-400">{t('address.delegations.contract.not_applicable')}</span>
+      </td>
+    );
+  }
+
+  return (
```

Comment:
I recommend returning `AddressCell` here so that it also includes the link / copy button automatically.